packages/launcher/package.json

```diff
@@ -36,15 +43,20 @@
     "@mainframe/rpc-handler": "^0.2.1",
     "@mainframe/toolbox": "*",
     "@mainframe/utils-id": "^0.1.0",
+    "@morpheus-ui/core": "0.0.8",
+    "@morpheus-ui/forms": "0.0.2",
+    "@morpheus-ui/fonts": "0.0.1",
+    "@morpheus-ui/icons": "0.0.1",
     "electron-store": "^2.0.0",
     "electron-util": "^0.10.2",
     "react": "^16.5.2",
     "react-art": "^16.5.2",
     "react-dom": "^16.5.2",
     "react-modal": "^3.6.1",
-    "react-native-web": "^0.9.1",
+    "react-native-web": "^0.9.9",
     "rxjs": "^6.3.3",
     "source-map-support": "^0.5.9",
+    "styled-components": "^4.1.2",
```

Comment:
Shouldn't `styled-components` be installed with `@morpheus-ui/core`?

Reply:
removed from packages.json